### PR TITLE
`init` should not move certain scripts #97

### PR DESCRIPTION
**Pull request by Miklet**
**What**: Leave npm lifecycle scripts and other core scripts untouched, convert `npm run` commansd to `nps`.

**Why**: To make migrating from npm to nps easier

**How**: 
* Added method which selects core scripts from package.json
* Moved stringify object method to separate module
* Added converting `npm run` scripts to `nps`

**Checklist**:

- [ ] Documentation N/A
- [x] Tests 
- [x] Ready to be merged 
- [ ] Added myself to contributors table  N/A

Hi @kentcdodds . My work is still in progress, tests will be added soon, but first want to make sure that my changes are suitable and also becasue at this point it is easier to revert some changes 😄

### CODE REVIEW

File: src/bin-utils/initialize/stringify-object.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import {isPlainObject} from 'lodash'
+
+function stringifyObject(object, indent) {
```
Comment (kentcdodds) [inline src/bin-utils/initialize/stringify-object.js]: Could we add a comment on this? I'm starting to forget what all this code is supposed to do 🙃

**Comment by Miklet** [comment]
Alright, I have rebased everything into one commit, added and updated tests, also added a comment as you wanted 😃 Now it is ready to merge in my opinion.

File: src/bin-utils/initialize/__tests__/index.js [hunk 2]
```diff
@@ -131,6 +131,33 @@ test(`initialize without any scripts should successfully create an empty package
   })
 })
 
+test('initialize without any core scripts and should not add any core scripts to package.json', () => {
```
Comment (Miklet) [inline src/bin-utils/initialize/__tests__/index.js]: Ahhh wrong test name, gonna fix it fast

Comment (Miklet) [inline src/bin-utils/initialize/__tests__/index.js]: Done!

**Pull request merged**